scripts/base/packet-protocols/ethernet/main.zeek

```diff
@@ -22,4 +13,8 @@ event zeek_init() &priority=20
 	PacketAnalyzer::register_packet_analyzer(PacketAnalyzer::ANALYZER_ETHERNET, 0x9100, PacketAnalyzer::ANALYZER_VLAN);
 	PacketAnalyzer::register_packet_analyzer(PacketAnalyzer::ANALYZER_ETHERNET, 0x8864, PacketAnalyzer::ANALYZER_PPPOE);
 	PacketAnalyzer::register_packet_analyzer(PacketAnalyzer::ANALYZER_ETHERNET, 0x8926, PacketAnalyzer::ANALYZER_VNTAG);
+
+	PacketAnalyzer::register_packet_analyzer(PacketAnalyzer::ANALYZER_ETHERNET, 0x0001, PacketAnalyzer::ANALYZER_SNAP);
+	PacketAnalyzer::register_packet_analyzer(PacketAnalyzer::ANALYZER_ETHERNET, 0x0002, PacketAnalyzer::ANALYZER_NOVELL_802_3);
+	PacketAnalyzer::register_packet_analyzer(PacketAnalyzer::ANALYZER_ETHERNET, 0x0003, PacketAnalyzer::ANALYZER_LLC);
```

Comment:
> Maybe it makes sense to introduce consts here and reuse them in the VLAN analyzer.

That sounds good.

> Also, how about using 1501-1503

I was going as low as I could just in case we have to add more of these later.

Reply:
> That sounds good.

Actually now that I think about this one more, it doesn't really make sense to re-use consts from the ethernet analyzer into the vlan analyzer. The two analyzers are distinct, and the values they use for these don't really mean anything between them. I could see defining a set of consts in both though.

Reply:
As 802.1Q uses the same values, i.e. Ethertypes, I think it would be okay to share. However, defining them in both analyzers is fine as well.